packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -28,15 +33,18 @@ interface AppHeaderProps {
   title: string;
   hasBackButton?: boolean;
   hasProfileButton?: boolean;
+  hasBurgerMenu?: boolean;
```

Comment:
Do you want to add a new story to the storybook?

Reply:
It's a bit trickier with the popups but I will try